### PR TITLE
Disable Chromium's poor MPRIS implementation

### PR DESCRIPTION
**Pull request by kytwb**
### Description

It prevents Chromium from hijacking media keys as per https://github.com/electron/electron/issues/18253#issuecomment-589542934.

### Motivation and Context

For https://github.com/getferdi/ferdi/issues/939.

I couldn't reproduce but propose to ship in the next nightly release for user testing.

### Checklist:

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`$ npm run lint`)
- [ ] I tested/previewed my changes locally

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,6 +60,9 @@ if (isWindows) {
   app.allowRendererProcessReuse = false;
 }
```
Comment (kris7t) [inline src/index.js:62]: I don't think we need this extra comment here.

**Review by kris7t — COMMENTED** [review]
Looks good to me. I tried it with on Linux with Skype for Web, and it seems to no longer steal my media keys.

File: src/index.js [hunk 2]
```diff
@@ -60,6 +60,9 @@ if (isWindows) {
...
 
+// 
+
+
```
Comment (kris7t) [inline src/index.js:65]: I don't think we need this extra comment here.

Comment (kytwb) [reply src/index.js:65]:
```suggestion 


```

**Pull request merged**